src/main/java/seedu/address/model/tag/Tag.java

```diff
@@ -16,7 +16,7 @@
 
     // Identity fields
     private final TagName tagName;
-    private final List<TagTask> tagTasks;
+    private List<TagTask> tagTasks;
```

Comment:
TaskClear currently mutates List<TagTask>. Discuss if this is acceptable or there should be another way to ensure that Tag's fields remain immutable is needed.